dist/style.css

```diff
@@ -0,0 +1,18 @@
+.nav-group-custom {
+  flex-direction: row-reverse;
+}
+
+.nav-item-custom {
+  margin: 0px 10px 0px 10px;
+}
+
+.nav-link-custom {
+  background-color: #0069D9;
+  border-radius: 5px;
+  color: white;
+}
+
+.nav-link-custom:hover {
+  background-color: rgb(0, 69, 143);
+  color: white;
+}
```

Comment:
Bootstrap has custom CSS classes for all of this I think
https://getbootstrap.com/docs/4.5/utilities/flex/
https://getbootstrap.com/docs/4.5/utilities/spacing/
https://getbootstrap.com/docs/4.5/utilities/colors/#background-color
https://getbootstrap.com/docs/4.5/utilities/colors/#color

And I think the `btn-primary` does the hover too

Reply:
Fixed!